dnn/variational_inference.ipynb

```diff
@@ -3,8 +3,8 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #33.      print('Test loss:', model.evaluate(x=sp.test.X, y=sp.test.Y, verbose=0))
</pre><p>If we save checkpoints, it may also be useful to print where a user can find the best checkpoint on their filesystem.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/161/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>